ui/lib/kmip/addon/templates/components/header-credentials.hbs

```diff
@@ -0,0 +1,26 @@
+<PageHeader as |p|>
+  <p.top>
+    <KmipBreadcrumb @scope={{@scope}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{@role}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+<div class="tabs-container box is-sideless is-fullwidth is-paddingless is-marginless">
+  <nav class="tabs">
+    <ul>
+      {{#link-to "credentials.index" @scope @role tagName="li"}}
+        {{#link-to "credentials.index" @scope @role}}
+          Credentials
+        {{/link-to}}
+      {{/link-to}}
+      {{#link-to "role" @scope @role tagName="li"}}
+        {{#link-to "role" @scope @role}}
+          Details
+        {{/link-to}}
+      {{/link-to}}
```

Comment:
If you have time, could you walk me through what is happening here? (re: the nested `link-to`s)

Reply:
So this is a sad thing we do because we were trying to use as much of bulma's CSS as we could initially. We'll get rid of this when we have our own tabs component.

But what's it's doing is constructing `<li><a/></li>` for the tab - `tagName` on the outer one is the important thing here. If I'm remembering correctly we needed to do this because bulma styles active tabs on the `li`, so we're using `link-to`'s ability to get the active class when the route matches.